src/main/scala/cromwell/engine/ExecutionStore.scala

```diff
@@ -27,7 +27,12 @@ class ExecutionStore(binding: WdlBinding) {
     for {
       callEntry <- table if callEntry._2 == ExecutionStatus.NotStarted
       call = callEntry._1
-      if call.prerequisiteCalls().forall(table.get(_).get == ExecutionStatus.Done)
+      if call.prerequisiteCalls().forall{
+        case c:Call => table.get(c).get == ExecutionStatus.Done
+        case x =>
+          println(s"runnableCalls: $x")
+          false
```

Comment:
This is a good example of the sort of HOF use that's bothering me (besides the ones w/ long blocks of code). `forall` is supposed to be a shortcut to see if a collection matches the shape of a simple predicate. In this example we have two separate predicates depending on the type of the contents *and* a side effect in the middle as well.